app/controllers/activity_comments_controller.rb

```diff
@@ -93,4 +92,18 @@ def report_id
   def comment_params
     params.require(:comment).permit(:body, :report_id)
   end
+
+  def prepare_trail_and_breadcrumb(is_new:)
```

Comment:
I'm finding this a little hard to parse - I wonder if it might be a bit clearer to opt for duplication here, and have a `prepare_new_trail_and_breadcrumb` and a `prepare_edit_trail_and_breadcrumb` method?

Also, are there already tests for this behaviour?

Reply:
I've split that method in two - looks better that way

I think the controller specs don't usually go into enough detail to cover trails/breadcrumbs. Can't see any existing tests in the specs I've touched referencing trails or breadcrumbs. Do you think we should add them, or leave the controller specs light?

Reply:
It looks like we've tested breadcrumbs in feature specs in the past e.g. [here](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/a74265fed7afd7ae1341aa05019d3e0c89d279dd/spec/features/users_can_search_for_activities_spec.rb#L31). It might be worth adding specs like that in the new/edit comments feature spec, if that's not too much of a pain, so at least we can catch things if they go missing.